src/test/java/com/wikia/webdriver/elements/mercury/components/Search.java

```diff
@@ -86,4 +78,70 @@ public Search clickClearSearchButton() {
     return this;
   }
 
+  public SearchResultsPage clickEnterAndNavigateToSearchResults() {
+    new Actions(driver).sendKeys(Keys.ENTER).perform();
+
+    return new SearchResultsPage();
+  }
+
+  public boolean isInputFieldSearchIconVisible() {
+    return isElementVisible(inputFieldSearchIcon);
+  }
+
+  public boolean isClearSearchButtonVisible() {
+    return isElementVisible(clearSearchButton);
+  }
+
+  public boolean isSearchInputFieldVisible() {
+    return isElementVisible(searchInput);
+  }
+
+  public boolean isSearchInputFieldEditable() {
+    try {
+      wait.forElementClickable(searchInput, 0).click();
+      return true;
+    } catch (NoSuchElementException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)